src/librustc_typeck/check/wfcheck.rs

```diff
@@ -141,7 +141,7 @@ impl<'a, 'gcx> CheckTypeWellFormedVisitor<'a, 'gcx> {
                 self.check_variances_for_type_defn(item, ast_generics);
             }
             hir::ItemUnion(ref struct_def, ref ast_generics) => {
-                self.check_type_defn(item, true, |fcx| {
+                self.check_type_defn(item, false, |fcx| {
```

Comment:
What's this argument and can it be removed from `check_type_defn`'s definition (i.e. are there any non-`false` calls)?

Reply:
The argument is called `all_sized` and is currently false for enums